packages/reporter-vue/src/store/reporter-store.ts

```diff
@@ -179,15 +179,19 @@ export const useReporterStore = defineStore({
 interface Runnable {
   readonly id: string,
   state: ComputedRef<RunnableState>
+  level: number
 }
 
-class Test implements Runnable {
+export interface Test {}
```

Comment:
Unused interface - potentially confusing, since this file exports both and interface and class with the same name, `Test`.